platforms/ide/ide-native/src/main/java/org/gradle/ide/visualstudio/internal/DefaultVisualStudioSolution.java

```diff
@@ -83,31 +80,13 @@ public SolutionFile getSolutionFile() {
     }
 
     @Override
-    @Internal
```

Comment:
This type is part of the mutable model and is no longer used (or should be used) as a @Nested bean.